config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,142 @@
+# Autoneg
+
+==================================================
+
+## NAME
+
+  autoneg
+
+## SYNOPSIS
+
+Autoneg solution uses network endpoint groups to provision load balancing across multiple clusters.
+This solution uses `./cluster/templates/autoneg.yaml` from [gke-autoneg-controller](https://github.com/GoogleCloudPlatform/gke-autoneg-controller).
+For demonstration purposes it uses a docker image with a simple Node.js service (bulankou/node-hello-world) with a single endpoint that prints out a message.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the autoneg folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/networking/helm/autoneg
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector
+1. [Helm](../../../README.md#helm)
+
+## USAGE
+
+All steps are run from this directory.
+
+1. Review and update the values in `./lb/values.yaml`. Note that if you change cluster name and locaiton, you will need to change how they are used in `gcloud container clusters get-credentials` commands below. [PROJECT_ID] refers to the project where all the GCP resources will be created.  
+
+1. Configure clusters and load balancing resources with Helm
+
+    ```bash
+    # validate your chart
+    helm lint ./lb/ --set projectId=[PROJECT_ID]
+
+    # check the output of your chart
+    helm template ./lb/ --set projectId=[PROJECT_ID]
+
+    # install your chart
+    helm install ./lb/ --set projectId=[PROJECT_ID] --generate-name
+    ```
+
+1. Wait for clusters to be created
+
+    ```bash
+    # The command uses cluster names based on the values passed in the previous step
+    kubectl wait --for=condition=Ready containercluster/cluster-na containercluster/cluster-eu
+    ```
+
+1. Configure first cluster
+
+    ```bash
+    # switch the context to the first cluster. The command uses cluster name and zone based on the values used to create the clusters.
+    gcloud container clusters get-credentials cluster-na --zone=us-central1-a
```

Comment:
Thanks. Because I added the disclaimer in the requirements - This solution assumes that all resources are installed in the same project, where the cluster with Config Connector is installed, and that load balancing resources are installed on the same cluster where Config Connector is installed, I'm leaving the more complex setup up the consumer to figure out.